VideoLocker/res/values-h600dp/dimens.xml

```diff
@@ -6,4 +6,7 @@
     <dimen name="edx_logo_height">140dp</dimen>
      <dimen name="social_margin_top">20dp</dimen>
      <dimen name="social_margin_bottom">15dp</dimen>
+    <dimen name="signup_social_title_margin">10dp</dimen>
```

Comment:
If this value is only used for the top margin, you might as well call this `signup_social_title_margin_top` to be consistent